sycl/test-e2e/Matrix/element_wise_all_ops_half.cpp

```diff
@@ -5,15 +5,12 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: aspect-fp16
-// REQUIRES: matrix
 // REQUIRES: matrix-fp16
+// REQUIRES: aspect-fp16
+// REQUIRES: aspect-ext_intel_matrix
```

Comment:
nit: keeping the order as much as possible and only changing 1 line would be easier to review.
```suggestion
// REQUIRES: aspect-fp16
// REQUIRES: aspect-ext_intel_matrix
// REQUIRES: matrix-fp16
```